### DISCUSSION TITLE
Mysterium VPN

### DISCUSSION THREAD

**Discussion by Piratealgeria** [privacytoolsIO/privacy-tools#5] [Submit Privacy Tools]
Open source, pay as you use with crypto, encrypted VPN
No registration required 
https://www.mysteriumvpn.com/
https://github.com/MysteriumNetwork

**Comment by privacytoolsIO** [comment]
thx, added here:
https://www.privacytools.io/dvpn

**Comment by nyabinary** [comment]
@privacytoolsIO add IVPN, which is great as well
